src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2162,6 +2162,22 @@ internal enum ErrorCode
         ERR_UnscopedRefAttributeInterfaceImplementation = 9102,
         ERR_UnrecognizedRefSafetyRulesAttributeVersion = 9103,
         ERR_BadSpecialByRefUsing = 9104,
+        ERR_BadExtensionUnderlyingType = 9105,
```

Comment:
>9105 [](http://example.com/codeflow?start=41&length=4)

Consider leaving a temporary gap in numbers (let's say starting with 9500) to simplify merges from main. The numbers could be packed right before merging into main